port_ocean/core/integrations/mixins/sync_raw.py

```diff
@@ -154,8 +154,8 @@ async def _register_resource_raw(
             )
             if not is_owner:
                 logger.info(
-                    f"Entity {entity_to_delete.identifier} hasn't been created by this integration, "
-                    f"skipping deletion..."
+                    f"Entity {entity_to_delete.identifier} either hasn't been created by this integration "
+                    f"or it doesn't exist, skipping deletion..."
```

Comment:
Skipping deletion of entity {entity_to_delete.identifier}, couldn't find entity that is related to integration.